### PR TITLE
Various skill decorations, engineer mine detonation cast finder and Arkk's anomalies as secondary target

### PR DESCRIPTION
**Pull request by Linkaaaaa**
Added decorations:
- Throw Mine
- Mine Field
- Throw Mine Detonation
- Mine Field Detonation
- Generic Mine Detonation (in case of precast out of combat)
- Hunter's Ward
- Symbol of Energy
- Symbol of Vengeance
- Symbol of Punishment
- Symbol of Blades
- Symbol of Resolution
- Stalwart Stand
- Shining River
- Scorched Aftermath

Instant Casts:
- Detonate Throw Mine
- Detonate Mine Field
- Generic Detonate (in case of precast)
- Magebane Tether

Other:
- Added generic Skill ID for Detonate when the source is unknown.
- Moved Temporeal Anomaly from Trash to Secondary Targets to check who damaged them and the health in Combat Replay. They show up only on selected subphases like the Enforcers on Dhuum.
- Added AddCircleSkillDecoration in ProfHelper and moved profession skill decoration to be using this API when appropriate.
- Fixed a bug on Sabir where the Flash Discharge SAK would conflict with other skill effects

### CODE REVIEW

File: GW2EIEvtcParser/EIData/ProfHelpers/Elementalist/ElementalistHelper.cs [hunk 1]
```diff
@@ -311,20 +312,18 @@ internal static void ComputeProfessionCombatReplayActors(AbstractPlayer player,
...
                 {
                     foreach (EffectEvent effect in meteorShowersMeteors)
                     {
-                        (long, long) lifespan = effect.ComputeLifespan(log, 1330);
+                        (long start, long end) lifespan = effect.ComputeLifespan(log, 1330);
                         var connector = new PositionConnector(effect.Position);
                         // -750 to make the decoration faster than in game
-                        replay.Decorations.Add(new CircleDecoration(180, (lifespan.Item2 - 750, lifespan.Item2), color, 0.5, connector).UsingFilled(false).UsingSkillMode(skill));
-                        replay.Decorations.Add(new CircleDecoration(180, (lifespan.Item2 - 750, lifespan.Item2), color, 0.2, connector).UsingFilled(false).UsingGrowingEnd(lifespan.Item2, true).UsingSkillMode(skill));
+                        replay.Decorations.Add(new CircleDecoration(180, (lifespan.start - 750, lifespan.end), color, 0.5, connector).UsingFilled(false).UsingSkillMode(skill));
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/ProfHelpers/Elementalist/ElementalistHelper.cs:325]: It was using Item2 before, now it uses what would have been Item1. Intentional?

**Comment by EliphasNUIT** [comment]
Any more changes or can I continue the review?

**Comment by Linkaaaaa** [comment]
You can continue :D

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**